src/git/install.sh

```diff
@@ -254,17 +266,7 @@ elif [ "${ADJUSTED_ID}" = "alpine" ]; then
     check_packages asciidoc curl-dev expat-dev g++ gcc openssl-dev pcre2-dev perl-dev perl-error python3-dev tcl tk xmlto
 
 elif [ "${ADJUSTED_ID}" = "rhel" ]; then
-
-    if [ $VERSION_CODENAME = "centos7" ]; then
-        check_packages centos-release-scl 
-        check_packages devtoolset-11
-        source /opt/rh/devtoolset-11/enable
```

Comment:
Was this removed because `VERSION_CODENAME` could have never been `centos7` due to the bug in https://github.com/devcontainers/features/pull/1056/files#diff-3dc3a1c6b68eed85dc6e030494337eeb7dacc6203f0db79595f4152878712508L29?

Reply:
Yes, exactly. Once I fixed the value of `$VERSION_CODENAME`, the tests failed attempting to install `devtoolset-11`.  I removed this to align with the previously accidentally-working behavior :slightly_smiling_face:.